packages/attestation-service/src/index.ts

```diff
@@ -33,6 +34,10 @@ async function init() {
     res.send(PromClient.register.metrics())
   })
   app.get('/status', rateLimiter, createValidatedHandler(StatusRequestType, handleStatusRequest))
+  app.get('/ready', rateLimiter, (_req, res) => {
+    res.send('Ready').status(200)
+  })
+  app.get('/healthz', rateLimiter, asyncHandler(handleLivenessRequest))
```

Comment:
typo in health?

Reply:
Apparently it's a common standard that originated from Google and is now used in popular open source projects like Kubernetes and Prometheus